content-service/server/database/utils.js

```diff
@@ -0,0 +1,53 @@
+const { shortIdToNum, numToShortId, } = require('../utils/shortid');
+const SHORT_URL = require('../../config.json').short_url;
+const nodeUrl = require('url');
+const resolveUrl = nodeUrl.resolve;
+const PARSED_SHORT_URL = Object.freeze(nodeUrl.parse(SHORT_URL));
+const nodePath = require('path');
+
+function shortUrlToId(url) {
+  const parsedUrl = nodeUrl.parse(url);
+  const isShortUrl =
+    parsedUrl.host === PARSED_SHORT_URL.host &&
+    parsedUrl.protocol === PARSED_SHORT_URL.protocol;
+
+  if (!isShortUrl) {
+    return false;
+  }
+
+  return nodePath.basename(parsedUrl.pathname);
+}
+
+function selectBeacons(knex) {
+  const st = require('knex-postgis')(knex);
```

Comment:
Should we require this outside of the `selectBeacons` function, also we do the same in the `beacons.js` file, and instead of passing st we do pass knex to require again the same object.